### ISSUE TITLE
Add link to metadata

### ISSUE THREAD

**Issue by jdleung** [vincentneo/CoreGPX#28]
I don't know if my way is correct:
```
        let href = GPXLink()
        href.href = "www.github.com"
        href.text = "Download link"
        let metadata = GPXMetadata()
        metadata.link = href
        root.metadata = metadata
```
The above will create an empty link in gpx file, the current codes in class GPXLink:

`        gpx.appendFormat("%@<%@%@>\r\n", indent(forIndentationLevel: indentationLevel), self.tagName())`

It shows correct by changing it to the following:
`        gpx.appendFormat("%@<%@%@>\r\n", indent(forIndentationLevel: indentationLevel), self.tagName(), attribute)`

BTW, I cannot `pod install` to a specified version, such as `pod 'CoreGPX', '0.4.2'`, I have to `pod update` every time.

**Comment by vincentneo** [comment]
I'll investigate this and update this to you later.

**Comment by vincentneo** [comment]
You are right. I added the `gpx.appendFormat("%@<%@%@>\r\n", indent(forIndentationLevel: indentationLevel), self.tagName(), attribute)` in the latest build.

Will be available in v0.4.4

Thanks for finding this bug out!

**Comment by vincentneo** [comment]
v0.4.4 has been pushed to CocoaPods.
